topics/climate/tutorials/pangeo-notebook/tutorial.md

```diff
@@ -0,0 +1,723 @@
+> > > # (see documentation on cartopy)
+> > > proj_plot = ccrs.Mercator()
+> > > 
+> > > # We need to project our data to the new projection and for this we use `transform`.
+> > > # we set the original data projection in transform (here PlateCarree)
+> > > p = dset.sel(time=slice(np.timedelta64(1,'h'),np.timedelta64(1,'D'))).sel(latitude=slice(43., 40.),
+> > >                                                                           longitude=slice(11.,15.))['pm2p5_conc'].plot(transform=ccrs.PlateCarree(),
+> > >                                                                                      vmin = 0, vmax = 35,
+> > >                                                                                      subplot_kws={"projection": proj_plot},
+> > >                                                                                      col='time', col_wrap=4,
+> > >                                                                                      robust=True,
+> > >                                                                                      aspect=dset.dims["longitude"] / dset.dims["latitude"],  # for a sensible figsize
+> > >                                                                                      cmap=cmc.roma_r)
+> > > # We have to set the map's options on all axes
+> > > for ax,i in zip(p.axes.flat,  (np.datetime64('2021-12-22') + dset.time.sel(time=slice(np.timedelta64(0),np.timedelta64(1,'D')))).values):
+> > >     ax.coastlines('10m')
+> > >     ax.set_title("CAMS PM2.5 " + pd.to_datetime(i).strftime("%d %b %H:%S UTC"), fontsize=12)
+> > > # Save your figure
+> > > plt.savefig("CAMS-PM2_5-fc-multi-Italy.png")
+> > > ```
+> > {: .code-in}
+> > 
+> >  ![PM2.5 over Italy](../../images/CAMS-PM2_5-fc-multi-Italy.png)
+> {: .solution }
+{: .question }
+
+
+> ### {% icon comment %}  `latitude=slice(47.3, 36.5)` and not `latitude=slice(36.5, 47.3)`
+> Why did we slice latitudes with `latitude=slice(47.3, 36.5)` and not `latitude=slice(36.5, 47.3)`?
+> - because when using slice, you need to specify values using the same order as in the coordinates. Latitudes are specified in 
+> decreasing order for CAMS.
+>
+{: .comment}
+
+
+## Where 
+
+- Sometimes we may want to make more complex selections with criteria on the values of a given variable and not only on its coordinates. For this we use `where`.
+- For instance, we may want to only keep PM2.5 if values are greater than 25 μm.m-3 (or any threshold you would like to choose)
+
+### Mask values that do not meet a criteria with `Where`
+
+```python
+print(dset.where(dset['pm2p5_conc'] > 25))
+```
+
+> ### {% icon comment %} What happened?
+> You may not see any changes but if you look carefuly at `pm2p5_conc` values, you will see many `nan`. In fact, we now have 
+> `nan` values whenever the criteria within the `where` statement is not met e.g. when PM2.5 <= 25
+>
+{: .comment}
+
+Let's plot one time to better see what happened:
+
+### Plotting with mask
```

Comment:
Is this title necessary? Perhaps it could be included inside the coding block.

Reply:
I will check. Thanks.